client/templates/includes/gillespyResultsEnsemble.pug

```diff
@@ -39,25 +39,25 @@ div#workflow-results-ensemble.card.card-body
       div
 
         h5.inline Plot Trajectories
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", data-hook="collapse-trajectories") +
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", id="collapse-trajectories", data-hook="collapse-trajectories") +
```

Comment:
The element on line 44 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value